apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -44,16 +48,19 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
   // Reference Id for the toast
   const toastId = useRef(null);
   // GraphQL mutation hook to create events
-  const [createEvent] = useMutation(CREATE_EVENT);
-  // Image of Event
-  const [imageFile, setImageFile] = useState(null);
-
+  const [createEvent] = useMutation<IEvent>(CREATE_EVENT);
+  const [uploadFile] = useMutation<{ uploadFile: IMedia }>(UPLOAD_PHOTO);
+  // Uploaded Image File
+  const [imageURLFile, setImageURLFile] = useState<String>(null); // Path of uploaded image on client, to be used in image previews
```

Comment:
```suggestion
  const [imageURLFile, setImageURLFile] = useState<string>(null); // Path of uploaded image on client, to be used in image previews
```
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object